libs/damap/src/lib/components/dashboard/dashboard.component.spec.ts

```diff
@@ -4,17 +4,28 @@ import { DashboardComponent } from './dashboard.component';
 import { MatButtonModule } from '@angular/material/button';
 import { MatIconModule } from '@angular/material/icon';
 import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { Router } from '@angular/router';
+import { RouterTestingModule } from '@angular/router/testing';
```

Comment:
`RouterTestingModule` is deprecated

```suggestion
import { Router, RouterModule } from '@angular/router';
```